neqo-transport/src/cc/classic_cc.rs

```diff
@@ -169,14 +183,52 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
         &mut self.pmtud
     }
 
+    fn detect_spurious_congestion_event(
```

Comment:
Given that `detect_spurious_congestion_event` is only called from `on_packets_acked`, does it need to be part of the `CongestionControl` trait, or would a simple `fn` on `ClassicCongestionControl` suffice?